test/unit_test_check_args.c

```diff
@@ -2,48 +2,48 @@
 
 int test_not_enough_args_1(void) {
   RESULT("test_not_enough_args_1", (EXP = 0) == (ACT = (int)check_args(0, NULL)));
-  return RES;
 }
 
 int test_not_enough_args_2(void) {
   RESULT("test_not_enough_args_2", (EXP = 0) == (ACT = (int)check_args(1, NULL)));
-  return RES;
 }
 
 int test_too_many_args(void) {
   RESULT("test_too_many_args_1", (EXP = 0) == (ACT = (int)check_args(3, NULL)));
-  return RES;
 }
 
 int test_invalid_path_1(void) {
-  char s1[] = "program";
-  char s2[] = "invalid.txt"; 
+  char s1[]   = "program";
+  char s2[]   = "invalid.txt"; 
   char *arr[] = { s1, s2 };
   RESULT("test_invalid_path_1", (EXP = 0) == (ACT = (int)check_args(2, arr)));
-  return RES;
 }
 
 int test_invalid_path_2(void) {
-  char s1[] = "program";
-  char s2[] = "text/input"; 
+  char s1[]   = "program";
+  char s2[]   = "txt/input"; 
   char *arr[] = { s1, s2 };
   RESULT("test_invalid_path_2", (EXP = 0) == (ACT = (int)check_args(2, arr)));
-  return RES;
 }
 
 int test_invalid_path_3(void) {
-  char s1[] = "program";
-  char s2[] = "mm@#!?(*\\m"; 
+  char s1[]   = "program";
+  char s2[]   = "mm@#!?(*\\m"; 
   char *arr[] = { s1, s2 };
   RESULT("test_invalid_path_3", (EXP = 0) == (ACT = (int)check_args(2, arr)));
-  return RES;
 }
 
 int test_null_path(void) {
-  char s1[] = "program";
+  char s1[]   = "program";
```

Comment:
What's with these spaces? I don't see why they're changed.
I don't particularly care, but it seems odd.